chpl/chpl-service/src/main/java/gov/healthit/chpl/scheduler/job/SplitDeveloperJob.java

```diff
@@ -288,6 +297,19 @@ private String createHtmlEmailBodyFailure(DeveloperDTO oldDeveloper, DeveloperDT
                 oldDeveloper.getName(),
                 newDeveloper.getName(),
                 ex.getMessage());
+        htmlMessage += createHtmlEmailFooter();
         return htmlMessage;
     }
+
+    private String createHtmlEmailFooter() {
```

Comment:
I think this footer is likely going to end up in emails coming from the system a lot going forward. Is there an email-util file somewhere that this could be moved to so we don't duplicate the effort?